bench-vortex/src/bin/clickbench.rs

```diff
@@ -69,12 +68,28 @@ fn main() {
         idempotent(&output_path, |output_path| {
             eprintln!("Downloading file {idx}");
             let url = format!("https://pub-3ba949c0f0354ac18db1f0f14f0a2c52.r2.dev/clickbench/parquet_many/hits_{idx}.parquet");
-            let r = reqwest::blocking::get(url)?.error_for_status()?;
-            let body = r.bytes()?;
+            let make_req = || reqwest::blocking::get(&url);
```

Comment:
I think if you keep the client alive instead of having one per request you might get a bit better behaviour still